src/components/WithinContentContext/context.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export interface WithinContentContextType {
+  withinContentContainer: boolean;
+}
+
+const WithinContentContext = React.createContext<WithinContentContextType>({
+  withinContentContainer: false,
```

Comment:
Not quite related to this PR but:

Can this be a boolean instead of an object? I can't see us ever needing to add extra values into this.

```
const WithinContentContext = React.createContext(false);
```